src/Components/pages/Host.jsx

```diff
@@ -176,27 +182,19 @@ class Host extends Component {
     }
 
     advanceQuestion() {
```

Comment:
method already exists in firebase-access (advanceDay)

Reply:
I basically am modifying some local properties as well (the local questionNum)